### ISSUE TITLE
Remember window size and position

### ISSUE THREAD

**Issue by libeclipse** [keybase/client#3324]
The GUI window when opened is the exact size of my screen vertically and positioned so that it's off the screen. It's fiddly trying to get it into a usable place.

It should open in a standard size in the middle of the screen, not the size of the display, and/or remember how the user left it last.

**Comment by michaelvolz** [comment]
Yes please, Remembering windows size and position would be really appreciated.
